app/models/concerns/submission_admin_concern.rb

```diff
@@ -63,9 +63,9 @@ def bare_druid
   end
 
   def augmented_dissertation_file_name
-    return unless dissertation_file.attached?
+    return unless augmented_dissertation_file.attached?
 
-    SignaturePageSupport.augmented_dissertation_path(dissertation_path: dissertation_file.filename.to_s)
+    augmented_dissertation_file.filename.to_s
```

Comment:
We could get the aug file's filename by getting the diss file and running gsub on it, but why not ask what the real aug file's filename is?